internal/postprocessor/main.go

```diff
@@ -363,7 +363,10 @@ func (c *config) getScopeFromGoogleapisCommitHash(commitHash string) (string, er
 	for _, filePath := range files {
 		for _, config := range generator.MicrogenGapicConfigs {
 			if config.InputDirectoryPath == filepath.Dir(filePath) {
-				scope := config.Pkg
+				// trim prefix
+				scope := strings.TrimPrefix(strings.TrimPrefix(config.InputDirectoryPath, "google/"), "cloud/")
```

Comment:
I am not sure we should use the inputdirectorypath here. There are other clients that come from other roots. For example the cloudbuild client comes from devtools: https://github.com/googleapis/googleapis/tree/master/google/devtools/cloudbuild

Reply:
In the example you are providing the InputDirectoryPath would be `"google/devtools/cloudbuild/v1"`, and after running this logic the scope would be `"devtools/cloudbuild"`. Is that not correct?

Reply:
nope, we trim off devtools and it becomes just cloudbuild: https://pkg.go.dev/cloud.google.com/go/cloudbuild.

I think there are a few other edge cases like this that make it hard to make assumptions off of the inputDir

Reply:
Okay is it safe to use those assumptions with ImportPath instead and trip `cloud.google.com/go` and the version?

Reply:
Used ImportPath